src/test/scala/org/zalando/kanadi/OAuthFailedSpec.scala

```diff
@@ -27,12 +26,20 @@ class OAuthFailedSpec extends Specification with FutureMatchers with Config {
     Subscriptions(nakadiUri, failingauthTokenProvider)
   val eventsClient = Events(nakadiUri, failingauthTokenProvider)
 
-  override def is: SpecStructure = s2"""
-    Call to subscriptions list should fail with invalid token   $oAuthCallSubscriptions
-    Call to publishEvents should fail with invalid token        $oAuthPublishEvents
-  """
-
-  def oAuthCallSubscriptions = Skipped("No way for current Nakadi docker image to detect \"wrong\" tokens")
-
-  def oAuthPublishEvents = Skipped("No way for current Nakadi docker image to detect \"wrong\" tokens")
+  ignore("No way for current Nakadi docker image to detect \"wrong\" tokens") { _ =>
```

Comment:
Scalatest's ignore properly supports having code contained inside which doesn't run unlike with specs2 so I have gone ahead and undid the removal of the test code which happened in the past.

Even though the tests don't work with the current Nakadi docker image, the code illustrates what the expected behaviour is which is useful (and as a bonus if hypothetical future Nakadi docker image ends up supporting this kind of test its trivial to remove the ignore).